assignment1.js

```diff
@@ -0,0 +1,69 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+
+
+function secondLargest(array) {
+ var i; var sh;
+var fh=array[0];
+if(array.length > 2) {
+          for(i=0;i<array.length;i++) {
+          if(array[i]>fh) fh=array[i];
+        }
+        sh=array[0];
+
+        for(i=0;i<array.length;i++) {
+          if(array[i]!=fh){
+            if(array[i]>sh) sh=array[i];
+          }
+        }
+}
+else  {
+  if(array[0]>array[1]) sh=array[1];
+  else sh=array[0];
```

Comment:
It is highly suggested to avoid the use of else, use only when you have a condition where you are 100% sure that you can do nothing around it.